src/common/common.module.ts

```diff
@@ -0,0 +1,4 @@
+import { Module } from '@nestjs/common';
```

Comment:
it is not necessary to create a new module for the common files, you can create a folder that is `/common `or `/shared` and there put everything that is shared between several modules

Reply:
Thanks Seba! I agree with you